src/averageAges.js

```diff
@@ -15,11 +15,17 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
   // learn how to use array methods like .filter .map .some .every .find .reduce
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+  const men = people.filter(person => (
+    person.sex === 'm' && (!century || century === Math.ceil(person.died / 100))
```

Comment:
hard to read If you a have a lot of conditions/comparing, you can create variable for `(!century || century === Math.ceil(person.died / 100))`